### ISSUE TITLE
Reduced-Motion: Generell langsam - weil die OS Einstellung nicht das Argument sein darf?!

### ISSUE THREAD

**Issue by deleonio** [public-ui/kolibri#3508]
Wir dürfen nicht davon ausgehen, dass alle diese OS-Einstellung kennen und die Motion reduzieren. Somit wäre es nicht gut, überhaupt schnelle Animationen anzubieten!

Ebenso gilt das für Computer, wo die Nutzenden die Einstellung gar nicht beeinflussen können.

**Comment by deleonio** [comment]
- [ ] Alle Animationen generell auf langsam setzen
- [ ] Reduce-Motion aus den Basis-CSS entfernen und Kommentar hinterlassen
- [ ] Langsame Animation für Accordions einbauen. Was machen wir, wenn Accordions viel Inhalt haben?

**Comment by Wolfram-Zell** [comment]
1) "Reduce motion" -> "Bewegung reduzieren" nicht "langsamer machen". Eine übliche Lösung ist die Animationszeit **kürzer** zu machen (<0.1s) und nicht notwendige/sinnvolle abzuschalten.
2) Einfach generell alle Animationen komplett zu entfernen ist aus UX-Sicht nicht sinnvoll.
3) Das würde bedeuten, dass wir **_alle_** BS/Browsereinstellungen ignorieren müssen und dass würde ich nicht so im vorbeifliegen in einem Ticket lösen. Gibt es hierzu weitere Quellen/Diskussionen? Falls ja, bitte Links einfügen.
4) Beim Accordion würde ich die Animation entfernen. (Die Animation ist nicht in Geschwindigkeit sondern in Dauer angegeben, Accordions mit viel Inhalt öffnen sich schneller.)
5) Animationen sind Darstellung, sollten also in den Themes definiert werden?

**Comment by deleonio** [comment]
Das Dev-Team hat eine andere Meinung! W3C

**Comment by Wolfram-Zell** [comment]
Wie ist der aktuelle Stand hierzu? Was ist durch wen zu überprüfen/implementieren? Was sind die nächsten Schritte, die wir diesbezüglich durchführen müssen?

**Comment by ElVariablo** [comment]
Vorgaben gibt es in der WCAG 2.1, Sc 2.3.3 

Ziel
Menschen mit vestibulären Störungen brauchen Kontrolle über Bewegungen, die durch Interaktionen ausgelöst werden. Nicht-essentielle Bewegung kann Reaktionen auf vestibuläre Störungen auslösen. Zu den Reaktionen der vestibulären (Innenohr-) Störung gehören Ablenkung, Schwindel, Kopfschmerzen und Übelkeit.

Anwendbarkeit
Immer anwendbar.

Was wird geprüft?
Durch Interaktion ausgelöste Bewegungsanimationen können deaktiviert werden, es sei denn, die Animation ist für die Funktionalität oder die übermittelten Informationen wesentlich.

Wie wird geprüft?
Prüfen, ob eine der drei Zustände gegeben ist:
- Keine unnötigen Animationen
- Nicht erforderliche Animationen können mit einem Steuerelement deaktiviert werden
- Die Funktionen zum Reduzieren von Bewegungen im User-Agent oder Betriebssystem werden unterstützt

**Comment by deleonio** [comment]
Meeting 04.07.2023 / BITV-Team

- Nach WCAG 2.1, Sc 2.2.2 - Pause, Hide, Stop - Keine Animationen länger als 5 Sekunden oder Pause-Schalter
  - für den Prüfschritt hat `reduced-motion` keine Auswirkung
  - CSS-Animation auf 5 s begrenzen
  - oder die Animation abschalten mit einem Schalter
  - Zentrale Webseiten-Einstellung - Animation abschalten
  - keine Loop-Animation
- Nach WCAG 2.1, Sc 2.3.1 Flackern
  - Animationen sind zwar erlaubt, aber generell keine **flackernde** Animationen verwenden
  - Flackern bedeutet mehr als 3 Farb- oder Musterwechsel pro Sekunde
  - `reduced-motion` ist keine Lösung!
  - kleiner 21 qPixel ;-)
- Nach WCAG 2.1, Sc 2.3.3 Animation from Interactions (Level AAA)
  -  ist die Technik `reduced-motion` (C39) erlaubt und kann eingesetzt werden.
  - Animationen können verwendet werden und müssen für Reduced-Motion entsprechend verlangsamt oder komplett auf 0 (aus) gesetzt werden.

**Comment by deleonio** [comment]
[👍](https://emojipedia.org/thumbs-up/)
